### ISSUE TITLE
Google CSS files should be stored on local

### ISSUE THREAD

**Issue by shashank-srikant** [shashank-srikant/sevasetu-mothercare#11]
Helps in offline development as well

**Comment by haptork** [comment]
yes, offline development is a benefit, however also consider the increase in latency due to no caching (google's css must already be cached) and because browsers can have limited number of connections with the same server and there are already local js, css and images to load.
